RecursiveExtractor/MiniMagic.cs

```diff
@@ -228,6 +232,10 @@ public static ArchiveFileType DetectFileType(Stream fileStream)
                 {
                     return ArchiveFileType.ISO_9660;
                 }
+                else if (buffer[0] == 'B' && buffer[1] == 'E' && buffer[2] == 'A' && buffer[3] == '0' && buffer[4] == '1')
```

Comment:
Do you have a sample UDF file to test with?

I think we may want the UDF check before the ISO check as well.

Reply:
Yep I have but it's malicious sample file from wild.  pls make sure to delete after extraction process. zip password: infected
[Document-2325.zip](https://github.com/microsoft/RecursiveExtractor/files/10451342/Document-2325.zip)

Reply:
Unfortunately I don't think a malicious sample woiuld be approrpiate to use for the test resources in the repo. Am working on trying to source a non-malicious UDF sample that works with the UDF Reader from DiscUtils.